zerver/forms.py

```diff
@@ -224,12 +225,58 @@ def validate(self, emails):
 
 class FindMyTeamForm(forms.Form):
     emails = MultiEmailField(
-        help_text="Add up to 10 comma-separated email addresses.")
+        help_text=_("Add up to 10 comma-separated email addresses."))
 
     def clean_emails(self):
         # type: () -> List[Text]
         emails = self.cleaned_data['emails']
         if len(emails) > 10:
-            raise forms.ValidationError("Please enter at most 10 emails.")
+            raise forms.ValidationError(_("Please enter at most 10 emails."))
 
         return emails
+
+class ProfileFieldForm(forms.ModelForm):
+    class Meta(object):
+        model = CustomProfileField
+        fields = ['name', 'field_type']
+
+def validate_custom_field_value(field, value):
+    # type: (CustomProfileField, Text) -> None
+    field_type = field.field_type
+    name = field.name
+    field_type_error = _("{name} does not have {field_type} type.")
+    if field_type == CustomProfileField.INTEGER:
```

Comment:
Yeah; it's worth checking if that works without regressing the format of the error messages, but it'd be nice to reuse that model.